x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/complete/complete.spec.ts

```diff
@@ -213,6 +213,43 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       });
     });
 
+    describe('LLM invocation with system message', () => {
+      let systemMessage: string;
+      before(async () => {
+        const { status, body } = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'GET /internal/observability_ai_assistant/functions',
+          params: {
+            query: {
+              scopes: ['all'],
+            },
+          },
+        });
+
+        expect(status).to.be(200);
+        systemMessage = body.systemMessage;
+      });
+
+      it('forwards the system message as the first message in the request to the LLM with message role "system"', async () => {
+        const interceptor = proxy.intercept('conversation', () => true);
+        observabilityAIAssistantAPIClient.editor({
+          endpoint: 'POST /internal/observability_ai_assistant/chat/complete',
+          params: {
+            body: {
+              messages,
+              connectorId,
+              persist: false,
+              screenContexts: [],
+              scopes: ['all'],
+            },
+          },
+        });
+        const simulator = await interceptor.waitForIntercept();
```

Comment:
You might have to call `await simulator.complete();` when you use `proxy.intercept`. That part happens automatically when using `completeAfterIntercept`.